content/doc/developer/testing/index.adoc

```diff
@@ -367,7 +367,19 @@ report.copyFrom(getClass().getResourceAsStream("lint-results_r20.xml"));
 ----
 
 ==== Customizing the `JENKINS_HOME` Directory
-TODO: Write this section.
+
+If you want to adjust the `JENKINS_HOME` within a test, use the `JenkinsRule`-methods link:https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html#withExistingHome(java.io.File)[`withExistingHome()`] or link:https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html#withNewHome()[`withNewHome()`].
+Let's assume in the directory "src/test/resources/home" everything is prepared for a good test suite, which you should use.
+This can be achieved with the following example:
+
+[source,java]
+----
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule().withExistingHome(new File("src/test/resources/home"));
+----
+
+If you need a fresh `JENKINS_HOME` you can use `withNewHome()` which creates a fresh home, which is by default what the `JenkinsRule()` at the beginning is.
```

Comment:
```suggestion
If you need a fresh `JENKINS_HOME`, you can use `withNewHome()` to create a fresh home, which `JenkinsRule()` at the beginning defaults to.
```
Small wording placement/adjustment for readability and to clarify the statement.